.github/workflows/CD.yaml

```diff
@@ -45,16 +45,13 @@ jobs:
       - uses: actions/checkout@v2
       - uses: actions/download-artifact@v2
```

Comment:
Es gibt hier keinen Grund für 2 Jobs. Wenn wir alles in einem Job machen sparen wir uns das Hoch- und Runterladen. und das zweite Checkout. Geht schneller und ist dann übersichtlicher.

Reply:
Habe ich versucht umzusetzen. Ich kann nur gerade nicht validieren ob es klappt, weil der CD-Workflow sich gerade irgendwie nicht manuell triggern lässt. (Pipeline überladen und es hängt in der Queue fest?)

Also bitte jemand validieren zu einem späteren Zeitpunkt bzw. ich validiere es später.

Reply:
ok